subsys/net/lib/lwm2m/Kconfig

```diff
@@ -273,25 +273,25 @@ config LWM2M_ENGINE_MAX_MESSAGES
 
 config LWM2M_COAP_BLOCK_SIZE
 	int "LWM2M CoAP block-wise transfer size"
-	default 256
+	default 512
```

Comment:
Just noticed that If this option has certain possible values, are we verifying it somewhere like having a build assert?

Reply:
Good point.
No, we are not verifying. I'll adjust the PR as I think this makes sense. Only certain values are possible.